CustomFileFormats/ResourcePacks.md

```diff
@@ -0,0 +1,114 @@
+# Dolphin Resource Pack Specification v3
```

Comment:
Any reason you didnt just update `docs/ResourcerPacks.md` instead?

Reply:
I saw `docs` but missed that `ResourcePacks.md` was in there.  Thanks!